### ISSUE TITLE
Alt text from uploaded files not making it onto the page

### ISSUE THREAD

**Issue by manez** [Islandora/documentation#1548]
An issue raised by @wgilling and confirmed during today's tech call: we require alt text when adding images as media, but it's not being served up on the node. 

@adam-vessey notes that: The image served is the “service file”, which doesn’t inherit the alt text. 

OpenSeadragon has its own issues with alt text, apparently, but even in that case we should be making it available somehow.

**Comment by seth-shaw-unlv** [comment]
The closest OpenSeadragon comes to providing alt-text is the complex-html overlay feature, which may be a tall order...

Another OpenSeadragon concern is site regression when JavaScript is unavailable/disabled which simply displays a black box. We should at least render out a link to the image, if not the image itself.
